ui/src/app/app.module.ts

```diff
@@ -13,13 +13,19 @@ import {ClarityModule} from 'clarity-angular';
 
 import {AppRoutingModule} from 'app/app-routing.module';
 import {AppComponent} from 'app/views/app/component';
+import {BugReportComponent} from 'app/views/bug-report/component';
 import {CohortBuilderComponent} from 'app/views/cohort-builder/search/cohort-builder/cohort-builder.component';
 import {CohortEditComponent} from 'app/views/cohort-edit/component';
 import {HomePageComponent} from 'app/views/home-page/component';
 import {SignInService} from 'app/services/sign-in.service';
 import {WorkspaceComponent} from 'app/views/workspace/component';
 import {WorkspaceEditComponent} from 'app/views/workspace-edit/component';
-import {CohortsService, WorkspacesService, Configuration, ConfigurationParameters, CohortBuilderService} from 'generated';
+import {CohortsService,
+  WorkspacesService,
+  BugsService,
+  Configuration,
+  ConfigurationParameters,
+  CohortBuilderService} from 'generated';
```

Comment:
What about a convention of breaking these before the column limit as many times as necessary instead of one break per import? My concern is that we're using a lot of vertical space for code that isn't particularly interesting.

Reply:
I agree it wastes space.

I tend do go this route if I have to wrap at all so that when I delete one item from the middle I don't need to re-wrap everything / the diff is easy to read. But I don't mind seeing either/both styles.

Reply:
Seems reasonable to not break these, especially if we aren't enforcing line limit on imports. Reverting that back.